tbx/core/blocks.py

```diff
@@ -740,6 +743,23 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/embed_block.html"
 
 
+class TypedTableBlock(blocks.StructBlock):
+    table = WagtailTypedTableBlock(
```

Comment:
WK has a line for `caption = blocks.CharBlock(required=False)` here - it doesn't look like we need that as I can see the caption in the Wagtail.

Reply:
I think I need to update kit - possibly with a wagtail upgrade the type table block has changed - I started with a caption field and found I had it twice.